Issue 3342 - RFE logconv.pl should have a replacement in CLI tools

jchapma opened the pull request:
Bug description:
	Perl DB is not available in RHEL 10 so the server access log
	analyzer tool (logconv.pl) needs to be ported to python.

Fix description:
	Initial draft of logconv.py, this is a work in progress. This
	commit message will be updated following code review/rework.

Fixes: https://github.com/389ds/389-ds-base/issues/3342

Reviewed by:

jchapma committed:
Issue 3342 - RFE logconv.pl should have a replacement in CLI tools

Bug description:
	Perl DB is not available in RHEL 10 so the server access log
	analyzer tool (logconv.pl) needs to be ported to python.

Fix description:
	Initial draft of logconv.py, this is a work in progress. This
	commit message will be updated following code review/rework.

Fixes: https://github.com/389ds/389-ds-base/issues/3342

Reviewed by:

jchapma committed:
separate group match from stats processing
logging, recommentations, other tidy up

progier389 reviewed (approved):
As agreed in team meeting - you can merge this first version to keep trace of it 
(knowing it will evolve)

mreynolds389 commented on ldap/admin/src/logconv.py:
```diff
@@ -0,0 +1,2888 @@
+        self.notesA = {}
+        self.notesF = {}
+        self.notesP = {}
+        self.notesU = {}
```
There is now a "notes=M" for MFA

jchapma committed:
style, bind dn report, time range updates, man page, few small bugs
Makefile.am, 389-ds-base.spec.in, more tidy up, removed connection group layer,
even more tidy up, processed time hours bug, file order bug
